timescaledb/overview/release-notes/index.md

```diff
@@ -31,18 +31,24 @@ while also empowering users with more control and flexibility to customize
 behaviors to suit your needs.  
 Some of these API updates are **breaking changes**.
 
-### What's new in TimescaleDB 2.4.1:
+### What's new in TimescaleDB 2.5:
 
-This release contains bug fixes since the 2.4.0 release. 
-We deem it high priority to upgrade since it is needed to support PostgreSQL 12.8 and 13.4.
+- Continuous aggregates for Distributed Hypertables
+- Support for Postgres 14
+- Experimental: Suport for timezones in 'time_bucket_ng()', including the 'origin' argument 
 
 
-The experimental features in the 2.4.0 release were:
+You can read more about this release on our [blogpost](TODO).
+This release contains bug fixes since the 2.4.1 release. 
+We deem it high priority to upgrade since it includes a correction for 'compress_policy'.
```

Comment:
```suggestion
We highly recommend that you upgrade as soon as possible, because this update includes a correction for a 'compress_policy' bug.
```

Reply:
Just curious, Engineering usually ranks the release between high, medium and low.
How would these 3 cases be in this new phrasing?

Reply:
I was trying to soften it from "next available opportunity". What about "when you are able to", or just "when you can"?